main/test/ca/uwaterloo/flix/language/feature/Test.Expression.NewChannel.flix

```diff
@@ -0,0 +1,175 @@
+/////////////////////////////////////////////////////////////////////////////
+// Tests the new Channel function that creates a new Channel               //
+//                                                                         //
+/////////////////////////////////////////////////////////////////////////////
+
+/************ Channels with default buffersize ***********/
+enum Color {
```

Comment:
In general we should avoid defining our own enums in test. Using Option, Result, List, etc. should be sufficient. (These are enums themselves).